test/intersection_tests.js

```diff
@@ -1,4 +1,4 @@
-var bezier = require('../lib/functions/bezier');
+/* var bezier = */ require('../lib/functions/bezier');
```

Comment:
could also remove the entire line

Reply:
I caught that in my other PR, but I've removed first in this one.